src/components/Alert/common.ts

```diff
@@ -5,7 +5,7 @@ export interface StatefulAlertProps {
      * Rivet alert styling. 
      * @see https://rivet.uits.iu.edu/components/overlays/alerts
      */ 
-    variant: "error" | "info" | "message" | "success",
+    variant: "error" | "danger" | "info" | "message" | "success",
```

Comment:
I think we should go ahead and remove the `error` variant as long as we mention it in the commit message for the release notes.